src/test/java/com/google/api/generator/gapic/composer/goldens/BUILD.bazel

```diff
@@ -0,0 +1,6 @@
+package(default_visibility = ["//visibility:public"])
+
+filegroup(
```

Comment:
Same as above - let's put this in an isolated directory outside of composer, e.g. `src/test/java/com/google/api/generator/gapic/dummy` or similar.

Reply:
SG, moved it.